### ISSUE TITLE
DOC Add LinkedIn button to the website (and remove Twitter)

### ISSUE THREAD

**Issue by TamaraAtanasoska** [fairlearn/fairlearn#1473]
#### Describe the issue linked to the documentation
The website currently still features a twitter button in the footer, but the account has been deleted for a few weeks already. The project has a LinkedIn page where we regularly post updates and educational information, but hasn't been added anywhere on the website.  

#### Suggest a potential alternative/fix
The PR resolving the issue should remove the Twitter icon from the footer and replace with an LinkedIn icon, and add LinkedIn to the header next to Github and Discord. (images below of the places where)

It might be that I have missed a place where this change needs to be applied as well, feel free to include a change if you find an appropriate place elsewhere.

Header:
<img width="1300" alt="Screenshot 2024-12-19 at 14 24 52" src="https://github.com/user-attachments/assets/ceb27ad9-ff51-42c0-b734-1dee79e6255d" />

Community part:
<img width="1436" alt="Screenshot 2024-12-19 at 14 34 32" src="https://github.com/user-attachments/assets/7c542c3e-4e00-4b1e-9735-bfc06878cc38" />

Footer:
<img width="1043" alt="Screenshot 2024-12-19 at 14 25 24" src="https://github.com/user-attachments/assets/8056280d-9ac6-4912-80d7-e6840bffdd20" />

**Comment by that-ar-guy** [comment]
hey @TamaraAtanasoska  can i take this one?

**Comment by TamaraAtanasoska** [comment]
> hey @TamaraAtanasoska can i take this one?

yes please! let me know if you have any questions along the process.

**Comment by TamaraAtanasoska** [comment]
Hi @that-ar-guy are you still interested? Let me know, otherwise this issue becomes free for grabs in a week from now. Thanks!

**Comment by VudhanthiNeeraja** [comment]
If this is up for grabs, can I take it?

**Comment by TamaraAtanasoska** [comment]
sure @VudhanthiNeeraja, go ahead!
